content/uk/docs/reference/glossary/pod-disruption-budget.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Обмеження переривання роботи Podʼів
+id: pod-disruption-budget
+full-link: /docs/concepts/workloads/pods/disruptions/
+date: 2019-02-12
+short_description: >
+ Обʼєкт, який обмежує кількість {{< glossary_tooltip text="Podʼів" term_id="pod" >}} реплікованого застосунку, які можуть бути вимкнені одночасно з причини добровільного переривання роботи.
+
+aka:
+ - PDB
+ - Pod Disruption Budget
```

Comment:
How about:
```yaml
title: PodDisruptionBudget
id: pod-disruption-budget
full-link: /docs/concepts/workloads/pods/disruptions/
date: 2019-02-12
short_description: >
 Обʼєкт, який обмежує кількість {{< glossary_tooltip text="Podʼів" term_id="pod" >}} реплікованого застосунку, які можуть бути вимкнені одночасно з причини добровільного переривання роботи.
aka:
 - PDB
 - Обмеження переривання роботи Podʼів
```

the same way that ReplicaSet etc are written with the API kind as the key.

Reply:
Ok. Agree

Reply:
```suggestion
title: PodDisruptionBudget
id: pod-disruption-budget
full-link: /docs/concepts/workloads/pods/disruptions/
date: 2019-02-12
short_description: >
 Обʼєкт, який обмежує кількість {{< glossary_tooltip text="Podʼів" term_id="pod" >}} реплікованого застосунку, які можуть бути вимкнені одночасно з причини добровільного переривання роботи.
aka:
 - PDB
```